terraform/azure.tf

```diff
@@ -634,3 +634,49 @@ resource "azurerm_sql_database" "sql-database" {
   depends_on          = ["azurerm_sql_server.sql-server"]
   tags {}
 }
+
+resource "random_string" "mysql_server" {
+  length  = 10
+  special = false
+  upper   = false
+}
+
+resource "azurerm_mysql_server" "mysql" {
+  name                = "${random_string.mysql_server.result}-mysqlsvr"
+  location            = "westeurope"
+  resource_group_name = "${azurerm_resource_group.rg.name}"
+
+  sku {
+    name     = "${var.sku_name}"
```

Comment:
If the variables being referenced here (in `variables.tf`) are not reused, we should simply hardcode the values in `azure.tf` until they're needed in multiple places. I dont expect we will need to reuse any the MySQL values.